libenkf/include/ert/enkf/queue_config.h

```diff
@@ -39,6 +39,7 @@ extern "C" {
 typedef struct queue_config_struct queue_config_type;
 
     queue_config_type * queue_config_alloc();
+    queue_config_type * queue_config_alloc_local_copy( queue_config_type * queue_config);
```

Comment:
The input argument should be `const queue_config_type` - always use `const` when possible,